Dockerfile

```diff
@@ -30,7 +30,7 @@ COPY CMakeLists.txt .
 COPY cmake cmake
```

Comment:
Can we restore the original Dockerfile here?

Reply:
Oops, will do!